src/js/angular/ttyg/templates/chat-view.html

```diff
@@ -0,0 +1,28 @@
+<link href="css/ttyg/chat-view.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+<div class="chat-view">
+    <div class="messages" ng-class="{'hidden': !selectedChat}">
+        <div id="messages-scrollable">
+            <div class="messages-hint text-muted" ng-if="!selectedChat.messages.length">
+                {{ 'ttyg.hint' | translate }}
+            </div>
+            <div ng-repeat="question in selectedChat.messages" class="m-1"
+                 ng-class="{'mt-3': question.role === 'user'}">
+                <span ng-if="question.role === 'user'">User:  </span>
```

Comment:
i18n

Reply:
This will be removed in a future request. It is currently used to mark that the message is from the user.